packages/fxa-payments-server/src/routes/Product/SubscriptionRedirect/index.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { render, cleanup, fireEvent, } from '@testing-library/react';
+import 'jest-dom/extend-expect';
+
+import {
+  SubscriptionRedirect,
+} from './index';
+
+afterEach(cleanup);
+
+it('performs a redirect to the expected URL for local product', () => {
+  assertRedirectForProduct('123doneProProduct', 'local', 'http://127.0.0.1:8080/');
+});
+
+it('performs a redirect to the default URL for unknown product', () => {
+  assertRedirectForProduct('beepBoop', 'bazquux', 'https://mozilla.org');
+});
+
```

Comment:
FWIW: Not testing the stage & dev server specific products here. Those IDs are kind of magic externally-sourced strings and I think just exercising local server and a default are enough to cover the logic.